index.php

```diff
@@ -24,5 +24,10 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
-require dirname(__FILE__).'/config/config.inc.php';
-Dispatcher::getInstance()->dispatch();
+use Symfony\Component\HttpFoundation\Request;
+
+require dirname(__FILE__) . '/config/config.inc.php';
+
+$kernel = new AppKernel(_PS_ENV_, true);
```

Comment:
I think if we handle this as is, there are several things that won't work:
- for starters the "front services" initialized in these files https://github.com/PrestaShop/PrestaShop/tree/47d2daba6e140c36d2894092d8c4a3de0c39ab3c/config/services/front won't be loaded Thus some services will not be accessible which is a breaking change
- this can be done thanks to our compiler pass like it's been done here for admin services https://github.com/PrestaShop/PrestaShop/blob/ff96226eae149d58b34ca6c5228b846595d05914/src/PrestaShopBundle/PrestaShopBundle.php#L64
- we need to replace the current custom Container with the symfony one, it means the reference in Context and Controller classes must be correctly filled:
  - for front controller https://github.com/PrestaShop/PrestaShop/blob/dc2c310af81a1c1a2b648462fd3007d3fc88e97e/classes/controller/FrontController.php#L2051
  - for context it's the only write access I could find so far (which is worrying ^^) https://github.com/PrestaShop/PrestaShop/blob/c13af1ac327d14f42e5a505eb3b46d3adc534eec/classes/Tools.php#L787
- We'll need a new parameter for our AppKernel, similar to environment (prod, dev, test) and debugMode (tru, false) we'll need a third one for front/admin (I don't know how to name it) This will allow us relying on the same kernel/container but we'll be able to adapt some services/configurations/routing differently depending on the provided environment
- then we should deprecate our `PrestaShop\PrestaShop\Adapter\ContainerBuilder` service and make sure it's not used anywhere anymore
- once all this is done we should check that the previous job of the legacy container is still covered by the new one (for example doctrine entities from modules, productcomments is a good module to test this)